soccer/src/soccer/strategy/agent/position/offense.cpp

```diff
@@ -559,4 +566,15 @@ rj_geometry::Point Offense::calculate_best_shot() const {
     }
     return best_shot;
 }
+
+// Checks whether ball is out of range for stealing/receiving
+bool Offense::ball_in_red(WorldState* last_world_state_) {
+    auto& ball_pos = last_world_state_->ball.position;
+    if (field_dimensions_.our_defense_area().contains_point(ball_pos) ||
+        field_dimensions_.their_defense_area().contains_point(ball_pos) ||
+        !field_dimensions_.field_rect().contains_point(ball_pos)) {
+        return true;
```

Comment:
GH actions (clang-tidy) has left a warning here. you should return the boolean literal directly instead.